README.md

```diff
@@ -621,12 +715,14 @@ Bivariance is among [the design decisions](https://github.com/Microsoft/TypeScri
 
 ## Flow's "mixed" type
 
-The TypeScript equivalent of the `mixed` type is simply:
+The TypeScript equivalent of the `mixed` type, in no strict mode, is simply:
```

Comment:
What do you mean by `no strict mode`?

Reply:
Maybe Flow `mixed` type can be represented better with Typescript `Object` type. What do you think?